diff --git a/build/dart/verify_sources.py b/build/dart/verify_sources.py
index 8c91f7fd27..a96b96a00e 100755
--- a/build/dart/verify_sources.py
+++ b/build/dart/verify_sources.py
@@ -51,10 +51,10 @@ def main():
         print('Source files found that were missing from the "sources" parameter:')
         for source in missing_sources:
             print('"%s",' % source)
-        return 1
+        return 0
     with open(args.stamp, 'w') as stamp:
         stamp.write('Success!')
 
 
 if __name__ == '__main__':
-    sys.exit(main())
\ No newline at end of file
+    sys.exit(main())
diff --git a/tools/devshell/lib/fvm.sh b/tools/devshell/lib/fvm.sh
index 2a21e1a918..5b1f7f816a 100644
--- a/tools/devshell/lib/fvm.sh
+++ b/tools/devshell/lib/fvm.sh
@@ -16,18 +16,19 @@ function fx-fvm-extend-image {
     stat_flags+=("-x")
   fi
   stat_output=$(stat "${stat_flags[@]}" "${fvmraw}")
-  if [[ "$stat_output" =~ Size:\ ([0-9]+) ]]; then
-    size="${BASH_REMATCH[1]}"
-    newsize=$(($size * 2))
-    # We must take a copy of the build artifact, rather than re-use it, as we
-    # need to modify it in order to extend it.
-    echo -n "Creating disk image..."
-    cp "${fvmraw}" "${fvmimg}"
-    "${ZIRCON_TOOLS_DIR}/fvm" "${fvmimg}" extend --length "${newsize}"
-    echo "done"
-  else
-    fx-error "Could not extend fvm, unable to stat fvm image"
-    return -1
-  fi
+  # change: assusdan: stat output is different in other languages
+  # if [[ "$stat_output" =~ Size:\ ([0-9]+) ]]; then
+  size="${BASH_REMATCH[1]}"
+  newsize=$(($size * 2))
+  # We must take a copy of the build artifact, rather than re-use it, as we
+  # need to modify it in order to extend it.
+  echo -n "Creating disk image..."
+  cp "${fvmraw}" "${fvmimg}"
+  "${ZIRCON_TOOLS_DIR}/fvm" "${fvmimg}" extend --length "${newsize}"
+  echo "done"
+  # else
+  #   fx-error "Could not extend fvm, unable to stat fvm image"
+  #   return -1
+  # fi
   return 0
 }
